website/templates/include/comment_pane_template.mako

```diff
@@ -24,14 +24,14 @@
                 <form class="form">
                     <div class="form-group">
                         <span>
-                            <textarea class="form-control" placeholder="Add a comment" data-bind="value: replyContent, valueUpdate: 'input', attr: {maxlength: $root.MAXLENGTH}"></textarea>
+                            <div class="form-control atwho-input" placeholder="Add a comment" data-bind="editableHTML: replyContent, attr: {maxlength: $root.MAXLENGTH}" contenteditable="true"></div>
```

Comment:
At https://github.com/CenterForOpenScience/osf.io/pull/5376#issuecomment-208498888 we say that "the user shouldn't be able to write html." In eb5e9c4251581e3b7aca176298b9e1db554a5d8f we protected against one HTML vector, copy/paste. Another vector for users to enter HTML is in using keyboard shortcuts. Seems like we should protect against that as well.

![mbn5r9qmvq](https://cloud.githubusercontent.com/assets/134455/16023034/df78fcd6-3188-11e6-9dbb-7a203c81c266.gif)